docker/archive/dest.go

```diff
@@ -19,17 +19,32 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
 		return nil, errors.Errorf("docker-archive: destination reference not supplied (must be of form <path>:<reference:tag>)")
 	}
-	fh, err := os.OpenFile(ref.path, os.O_WRONLY|os.O_EXCL|os.O_CREATE, 0644)
+	// tries to open a file first to accomodate for load when using stdout
+	// as the file is created first then written to
+	// if the file doesn't exist then it creates a new file for exclusive use
+	// if the file exists, it checks to ensure the size is 0
```

Comment:
Right, I meant it works for multiple tags of the same image. So if we save 
alpine:2.6
alpine:2.7
into the same docker archive and load that using kpod, it will load both alpine:2.6 and alpine:2.7

The error you are getting above is because the loadTarManifest was throwing an error if it received more than one object. I changed that in containers/image to not throw an error if there is more than one so that kpod load can load all the tags in one archive.
This was the pull request for that change that got accepted and merged https://github.com/containers/image/pull/309